src/services/analytics.ts

```diff
@@ -664,6 +670,39 @@ type RemixHide = {
   handle: string
 }
 
+type ClaimAudioRequest = {
+  eventName: Name.CLAIM_AUDIO_REQUEST
+  wallet: string
```

Comment:
nit: I *think* we have a wallet or address type for this somewhere, could be wrong though

Reply:
nice!